### ISSUE TITLE
set namespace via header value

### ISSUE THREAD

**Issue by c-nv-s** [kelindar/folio#1]
placing a folio application behind oauth2proxy (https://github.com/oauth2-proxy/oauth2-proxy) is a quick way to get user information from an existing internal authentication system.

the oauth2proxy authenticates the user and passes the relevant user information to the downstream application via header values ``X-Forwarded-User``, ``X-Forwarded-Groups``, ``X-Forwarded-Email`` (https://oauth2-proxy.github.io/oauth2-proxy/configuration/overview#header-options)

so it might be nice if there is a setting where the user can optionally choose which header value to use from the request to set the namespace

**Comment by kelindar** [comment]
Yeah, this was actually the plan. Just haven't gotten to adding a user model yet, this project is still in it's very early stage.

**Comment by c-nv-s** [comment]
Happy new year! just FYI still really keen on this feature so wondered if there was any update?

**Comment by c-nv-s** [comment]
@kelindar ?

**Comment by kelindar** [comment]
@c-nv-s Unfortunately I'm super busy at the moment, for a next few months at the very least. Would you be able to contribute this?

**Comment by c-nv-s** [comment]
ah ok understood... I'm also busy working on a project at the moment so wouldn't be able to either.
